### DISCUSSION TITLE
Theme Page Transitions

### DISCUSSION THREAD

**Discussion by AL-THE-BOT-FATHER** [flet-dev/flet#806] [General]
First, thank you for Flet. I am thinking about making it my "go to" for making GUIs. 

I have a quick question about Themes and Page Transitions. I understand the Flet supports 4 transition types: 
FADE_UPWARDS, OPEN_UPWARDS, ZOOM, and CUPERTINO.

My question is: is there a way to have page transitions without the animations? I want no animation between the transition between views. 

I tried: 

**theme = Theme()
theme.page_transitions.android = None
theme.page_transitions.ios = None
theme.page_transitions.macos = None
theme.page_transitions.linux = None
theme.page_transitions.windows = None
page.theme = theme
page.update()**

Any help would be greatly appreciated!

-Al

**Comment by FeodorFitsner** [comment]
Created a new issue: https://github.com/flet-dev/flet/issues/808
Will implement it shortly.
